### PR TITLE
Add skaffold triage party config to skaffold repo

### PR DESCRIPTION
**Pull request by tejal29**
Adding triage party config and deploy scripts to skaffold repo so that any changes made to the script and config are reviewed by the team and stay here.

Next Steps
- [x] Update internal docs to use this script. 

Thanks
Tejal

### CODE REVIEW

File: deploy/triage-party/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# limitations under the License.
+
+set -eux
+
+# Export this environment variable before running this script
+echo "token path: ${GITHUB_TOKEN_PATH}"
+
+export PROJECT=k8s-skaffold
+IMAGE="gcr.io/k8s-skaffold/teaparty:$(date +%F-%s)"
+export
```
Comment (briandealwis) [inline deploy/triage-party/deploy.sh:24]: I think you mean this:
```suggestion
export IMAGE="gcr.io/k8s-skaffold/teaparty:$(date +%F-%s)"
```

**Review by briandealwis — COMMENTED** [review]
Is this PR missing the Dockerfile?

File: deploy/triage-party/deploy.sh (hunk 1 above)
Comment (tejal29) [reply deploy/triage-party/deploy.sh:24]: The dockerfile is this https://github.com/google/triage-party/blob/master/Dockerfile. 

Let me actually point to latest teaparty image and in the internal docs ad an extra step to publish this image.

Comment (tejal29) [reply deploy/triage-party/deploy.sh:24]: nvm. I blindly copied the deploy script from  https://github.com/google/triage-party. 

I will change the `docker build -t` etc.

Comment (tejal29) [reply deploy/triage-party/deploy.sh]: I decided to delete the `deploy.sh` instead change the internal docs to point to skaffold config in the skaffold repo instead the one in the http://go/skaffold-triage-party-deploy

**Pull request merged**